recipes/phyx/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "phyx" %}
+{% set version = "0.99.0.20180601" %}
+{% set hash = "e4a698b0c51988398f61c72a2cf1df156352481345cf7867fc1ce12072e1f6b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FePhyFoFum/{{ name|lower }}/archive/84634ef7ea874cea98f88694ad2e0f95836164ac.tar.gz
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc
```

Comment:
we should also use the new `compiler` here and the host section

Reply:
I will update once we have a new tagged release. I will remind them.